backup.py

```diff
@@ -1,207 +1,195 @@
+#!/usr/bin/python3
+
 import os
 import sys
 import datetime
-
+import boto3
+import click
+import logging
+from boto3.exceptions import S3UploadFailedError
 from subprocess import call
+from colorama import init
+from termcolor import colored
 
-import click
+init()
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+DEFAULT_REGION='us-west-2'
```

Comment:
I'd go with `'us-east-1'` as `DEFAULT_REGION` as that's what I generally see elsewhere.

Reply:
That sounds agreeable.